File: ITManagementServer/src/main/java/models/responseModels/departments/GetFullDepartmentInfoResponseModel.java
package models.responseModels.departments;

import models.responseModels.workers.WorkerGridResponseModel;

import java.util.ArrayList;

public class GetFullDepartmentInfoResponseModel {
    private int DepartmentId;
    private String Title;
    private String WorkerDuties;
    public ArrayList<WorkerGridResponseModel> Workers;

    public GetFullDepartmentInfoResponseModel() {
        Workers = new ArrayList<>();
    }

    public int getDepartmentId() {
        return DepartmentId;
    }

    public void setDepartmentId(int departmentId) {
        DepartmentId = departmentId;
    }

    public String getTitle() {
        return Title;
    }

    public void setTitle(String title) {
        Title = title;
    }

    public String getWorkerDuties() {
        return WorkerDuties;
    }

    public void setWorkerDuties(String workerDuties) {
        WorkerDuties = workerDuties;
    }

    public ArrayList<WorkerGridResponseModel> getWorkers() {
        return Workers;
    }

    public void setWorkers(ArrayList<WorkerGridResponseModel> workers) {
        Workers = workers;
    }
}

File: ITManagementClient/Models/ResponseModels/Departments/GetFullDepartmentInfoResponseModel.cs
using System.Collections.Generic;
using ITManagementClient.Models.ResponseModels.Workers;

namespace ITManagementClient.Models.ResponseModels.Departments
{
    public class GetFullDepartmentInfoResponseModel
    {
        public int DepartmentId { get; set; }

        public string Title { get; set; }

        public string WorkerDuties { get; set; }

        public IEnumerable<WorkerGridResponseModel> Workers { get; set; }
    }
}